### PR TITLE
Fix setting of BatchNormalization config.

### PR DESCRIPTION
**Pull request by hgaiser**
The configuration of the batch normalization layers wasn't stored correctly, because the super class was not called. This causes parameters like `name` or `epsilon` to be lost when loading a previously saved model.

### CODE REVIEW

**Comment by 0x00b1** [comment]
Thanks, @hgaiser!

**Comment by 0x00b1** [comment]
Thanks, @hgaiser!
